src/main/java/org/testng/IModule.java

```diff
@@ -0,0 +1,11 @@
+package org.testng;
+
+import com.google.inject.Module;
+
+/**
+ * This is a marker interface for a Guice module to be instantiated with {@link java.util.ServiceLoader}.
+ *
+ * @author Bartosz Popiela
```

Comment:
Same goes here as well. Please have the `@author` tag removed.

Reply:
Done.